bundles/org.openhab.binding.electroluxappliance/README.md

```diff
@@ -120,3 +154,28 @@ Switch electroluxapplianceIonizer "Electrolux Air Ionizer Setting" {channel="ele
 // Safety Lock
 Switch electroluxapplianceSafetyLock "Electrolux Air Safety Lock Setting" {channel="electroluxappliance:air-purifier:myAPI:myair-purifier:safetyLock"}
 ```
+
+##  `demo.items` Example - Portable Air Conditioner
+
+```java
+Group Electrolux_Air_Conditioner "Electrolux Air Conditioner" [AirConditioner]
+Number:Temperature Electrolux_Air_Conditioner_Ambient_Temperature "Ambient Temperature [%.1f %unit%]" <Temperature> (Electrolux_Air_Conditioner) [Measurement, Temperature] { channel="electroluxappliance:portable-air-conditioner:myAPI:myportable-air-con:ambient-temperature", unit="°C" }
+Switch Electrolux_Air_Conditioner_Child_Lock "Child Lock" <Lock> (Electrolux_Air_Conditioner) [Mode, Status] { channel="electroluxappliance:portable-air-conditioner:myAPI:myportable-air-con:child-ui-lock" }
+Switch Electrolux_Air_Conditioner_Compressor_Running "Compressor Running" <Switch> (Electrolux_Air_Conditioner) [Mode, Status] { channel="electroluxappliance:portable-air-conditioner:myAPI:myportable-air-con:compressor-state" }
+Switch Electrolux_Air_Conditioner_Evap_Defrost_State "Evap Defrost State" <Switch> (Electrolux_Air_Conditioner) [Mode, Status] { channel="electroluxappliance:portable-air-conditioner:myAPI:myportable-air-con:evap-defrost-state" }
+String Electrolux_Air_Conditioner_Fan_Speed "Fan Speed" <Flow> (Electrolux_Air_Conditioner) [Mode, Status] { channel="electroluxappliance:portable-air-conditioner:myAPI:myportable-air-con:fan-mode" }
+Switch Electrolux_Air_Conditioner_Fan_Swing "Fan Swing" <Flow> (Electrolux_Air_Conditioner) [Mode, Status] { channel="electroluxappliance:portable-air-conditioner:myAPI:myportable-air-con:fan-swing" }
+Switch Electrolux_Air_Conditioner_Four_Way_Valve_State "Four Way Valve State" <Switch> (Electrolux_Air_Conditioner) [Mode, Status] { channel="electroluxappliance:portable-air-conditioner:myAPI:myportable-air-con:fourway_valve-state" }
```

Comment:
```suggestion
Switch Electrolux_Air_Conditioner_Four_Way_Valve_State "Four Way Valve State" <Switch> (Electrolux_Air_Conditioner) [Mode, Status] { channel="electroluxappliance:portable-air-conditioner:myAPI:myportable-air-con:fourway-valve-state" }
```

Reply:
Corrected now and pushed with previous PR comment.